website/src/pages/dictionary/index.tsx

```diff
@@ -63,6 +63,8 @@ import Dictionary from '../../components/Dictionary';
 import { createSchemasWithDiffs, getDictionary, getDictionaryDiff } from '../../helpers/schema';
 import { ChangeType, Schema } from '../../../types';
 
+const FF_COMPARISON = process.env.FF_COMPARISON === 'true' ? true : false;
```

Comment:
do we have a centralized config file to host all the envs?

Reply:
we do not. can add.